Standards/WCAG2AAA/Sniffs/Principle4/Guideline4_1/4_1_2.js

```diff
@@ -199,7 +199,7 @@ _global.HTMLCS_WCAG2AAA_Sniffs_Principle4_Guideline4_1_4_1_2 = {
             var requiredValue = requiredValues[nodeName];
 
             // Any element that doesn't have specific handling must have content.
-            if (!matchingRequiredNames) {
+            if (!matchingRequiredNames && nodeName !== 'input_hidden') {
```

Comment:
instead of doing this, shouldn't you add 'input_hidden' to the requiredNames variable above on line 159?